### PR TITLE
D3ASIM-2829: Adding current market fee for dso_market_stats

### PR DESCRIPTION
**Pull request by faizan2590**

### CODE REVIEW

**Review by Hassan754 — DISMISSED** [review]

File: src/d3a/models/area/stats.py [hunk 1]
```diff
@@ -132,6 +132,8 @@ def get_last_market_stats(self, dso=False):
             self.current_market.time_slot, default_trade_stats_dict))
         out_dict[current_market_time_slot_str]["market_bill"] = \
             self._get_current_market_bills()
+        out_dict[current_market_time_slot_str]["market_fee"] = \
```
Comment (hannesdiedrich) [inline src/d3a/models/area/stats.py:135]: In order to not get confused with the grid_fee setting, i suggest to rename this to something like:
`earned_market_fee` or `market_fee_revenue`

Comment (faizan2590) [reply src/d3a/models/area/stats.py:135]: Hi @chrisd12 , would you mind suggesting which one those names would be better in terms of API simplification, `market_fee , earned_market_fee  or market_fee_revenue `?

Comment (chrisd12) [reply src/d3a/models/area/stats.py:135]: @faizan2590 I think **market_fee_revenue** is the most comprehensive name for API users. One question : is it the cumulative revenue over the simulation or the exact revenue of that specific market slot ?

Comment (faizan2590) [reply src/d3a/models/area/stats.py:135]: @chrisd12 Its not the cumulative revenue but the exact revenue during that specific market slot.

**Review by hannesdiedrich — APPROVED** [review]
LGTM

File: src/d3a/models/area/stats.py (hunk 1 above)
Comment (chrisd12) [reply src/d3a/models/area/stats.py]: @faizan2590 ok great perfect !

**Review by Hassan754 — APPROVED** [review]

**Pull request merged**